README.md

```diff
@@ -37,3 +35,4 @@ Pgbouncer won't run without the following environment variables passed in:
   - These will be encrypted in future.
   - A default "pw" is provided if the default `$PGB_ADMIN_USERS` value is used, but otherwise no default is provided.
 
+    1. `docker run -rm --name pgbouncer:local`
```

Comment:
This addition needs some context/explanation. It's not clear why it's here.

Reply:
Not sure what this is, but it's been removed